packages/blade/src/components/Breadcrumb/Breadcrumb.web.tsx

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable react-native-a11y/has-valid-accessibility-role */
+import React from 'react';
+import type { BreadcrumbProps } from './types';
+import { BreadcrumbContext } from './BreadcrumbContext';
+import BaseBox from '~components/Box/BaseBox';
+import { Text } from '~components/Typography';
+import { makeAccessible } from '~utils/makeAccessible';
+import { getStyledProps } from '~components/Box/styledProps';
+import { metaAttribute, MetaConstants } from '~utils/metaAttribute';
+
+const Separator = ({
+  size,
+  color,
+}: Pick<BreadcrumbProps, 'size' | 'color'>): React.ReactElement => {
+  return (
+    <Text
+      as="span"
+      color={color === 'white' ? 'surface.text.staticWhite.muted' : 'surface.text.gray.muted'}
+      size={size}
+      weight="medium"
+    >
+      /
+    </Text>
+  );
+};
+
+const listStyleNone = { listStyle: 'none' };
+
+const Breadcrumb = ({
+  size = 'medium',
+  color = 'primary',
+  showLastSeparator = false,
+  accessibilityLabel = 'Breadcrumb',
+  children,
```

Comment:
add testID also?

Reply:
🤔 not needed, people can do getByRole(nav)

Reply:
does that work with cypress/playwright too?

Reply:
yes, https://playwright.dev/docs/locators